userdocs/src/installation.md

```diff
@@ -351,6 +159,7 @@ system settings.
 eksctl completion powershell > C:\Users\Documents\WindowsPowerShell\Scripts\eksctl.ps1
 ```
 
+<!-- Todo: Move features to homepage-->
```

Comment:
Is this TODO meant to be resolved in this PR?

Reply:
this will be a future iteration, we need to come up with a design for this